src/elements/content-sidebar/MetadataInstanceEditor.tsx

```diff
@@ -1,48 +1,44 @@
-import {
-    MetadataInstanceForm,
-    type MetadataTemplateInstance,
-    UnsavedChangesModal,
-    type MetadataTemplate,
-} from '@box/metadata-editor';
+import { AutofillContextProvider, MetadataInstanceForm, type MetadataTemplateInstance } from '@box/metadata-editor';
+import { JSONPatchOperations } from '@box/metadata-editor/types/lib/components/metadata-instance-editor/subcomponents/metadata-instance-form/types';
+import { FormValues } from '@box/metadata-editor/types/lib/components/metadata-instance-editor/types';
```

Comment:
are neither one of these exported at the root level of this package?

Reply:
not yet, but I saw we have one PR to export `JSONPatchOperations` so will update that when ready

Reply:
Those types can be imported directly from metadata-editor package. Just make sure you're using newest version